### ISSUE TITLE
Simplify SecKey.attributes codepaths?

### ISSUE THREAD

**Issue by feralbob** [outfoxx/Shield#10]
Hi Everyone,

I'm still a newbie at Cryptography; especially with the iOS/Mac SDKs.

Looking at `project.yml` - I see the deployment targets are set for at least iOS 10 and Mac 10.12

```
deploymentTarget:
      macOS: 10.12
      iOS: 10.0
      tvOS: 10.0
      watchOS: 3.0
```

Given the deploymentTarget above can the `attributes` function in `SecKey.swift` be simplified to just use `SecKeyCopyAttributes` no matter what codepath?

I'll prepare a simple PR anyway - but I'd welcome the teams thoughts with me being a newbie. Looking forward to hearing what you think and if I'm completely off the mark or not :)

Thanks

**Comment by feralbob** [comment]
Oh I just spotted that the first 
`#available` call checks for WatchOS 3.3 - SecKeyCopyAttributes looks to be available from 3.0 onwards. Was there a specific reason the 3.3 check was made?

**Comment by kdubb** [comment]
Fixed in 703ba2140696aea97091dfbf6c44ac9ab87352f7
